cmd/metrics.go

```diff
@@ -81,9 +92,10 @@ func (c *minioCollector) Describe(ch chan<- *prometheus.Desc) {
 func (c *minioCollector) Collect(ch chan<- prometheus.Metric) {
 
 	// Expose MinIO's version information
-	minioVersionInfo.WithLabelValues(Version, CommitID).Set(float64(1.0))
+	minioVersionInfo.WithLabelValues(Version, CommitID).Set(1.0)
 
 	storageMetricsPrometheus(ch)
+	nodeHealthMetricsPrometheus(ch)
```

Comment:
Shouldn't this be part of v2 metrics? Why change existing metrics when we are adding the v2.

Reply:
We can drop it from the current metrics but for existing customers who have not yet moved over to v2, this metric might still be useful to add.

Reply:
I see, we can keep it in old metrics as well. It isn't a breaking change, so should be fine.